index.d.ts

```diff
@@ -18,6 +18,7 @@ export function scrollToElement(elem: ElementFinder, timeoutInMs?: number): Prom
 export function setTimeout(timeoutInMs?: number): void
 
 // waiters
+export function waitForAlertToBePresent(timeoutInMs?: number): void
```

Comment:
```suggestion
export function waitForAlertToBePresent(timeoutInMs?: number): Promise<T>
```

See http://www.protractortest.org/#/api?view=ProtractorExpectedConditions.prototype.alertIsPresent